src/components/welcome/welcome.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react'
+
+const Welcome = (props) => {
+  const { welcomeString } = props
+  return (
+    <h1 data-testid='welcome'>{welcomeString}</h1>
```

Comment:
I like the use of a tag other than "class" within your `h1` element, but I don't like the name "data-testid" since you are using the word "test" in environments like production. I would just recommend "data-id"